CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added limited Fuchsia support (#[1285](https://github.com/nix-rust/nix/pull/1285))
 - Added `getpeereid` (#[1342](https://github.com/nix-rust/nix/pull/1342))
 ### Fixed
+- Removed s390x exemptions from `nix::sys::statfs` `MAGIC` constants, since the corresponding types in `libc` have been corrected.
```

Comment:
How about "Define *_MAGIC constants on Linux s390x", and include a PR link?

Reply:
Sure! Have updated accordingly.